.travis.yml

```diff
@@ -11,5 +11,4 @@ install:
   - pip freeze
 # command to run tests
 script:
-    - flake8 .
-    - mypy csob_paymentgateway
+    - flake8 csob
```

Comment:
Vypadá to že travis není nastaven pro tohle repo, mám to zapnout?

Reply:
to by bylo fajn

Reply:
už by mělo být zapnuto